### PR TITLE
Assignment-1 

### PR DESCRIPTION
**Pull request by DaisyPalencia**
Paired with @j0shh3ss

### CODE REVIEW

File: wordle-web/src/views/AboutView.vue [hunk 1]
```diff
@@ -0,0 +1,23 @@
+<template>
```
Comment (Malcolm-Shepherd) [inline wordle-web/src/views/AboutView.vue:1]: This page meets all the technical requirements, but they wanted it to be named game.vue

**Review by Malcolm-Shepherd — COMMENTED** [review]
## Instructions

1. Create a Vue app ✅
   - Uses TypeScript ✅
   - Add Vuetify ✅
   - Uses Vitest ✅

3. Create a simple page called game.vue ❌
   - Page should have a card with a heading that with a clever name for the game ✅
   - some default placeholder body text ✅
   - and a button to take you back to the home page ✅
   - there should be a button on the home page to access this page ✅

4. Check all the code into your repo fork ✅

5. Create a static web app in Azure ✅
   - Create the container where your GitHub repo will deploy
   - Link the site with your GitHub repo

6. Update the CI/CD pipeline ✅
   - Pipeline runs on each check in
   - Vitest run and pass ✅
   - Vue app builds ✅
   - Vue app deploys to Azure ✅

7. Azure site works ✅
   - Post your Azure URL in your readme.md file. Replace the URL in the readme with your URL. ✅

File: wordle-web/src/views/AboutView.vue (hunk 1 above)
Comment (DaisyPalencia) [reply wordle-web/src/views/AboutView.vue:1]: Thank you, I changed the name to game.vue.

File: wordle-web/src/views/game.vue [hunk 2]
```diff
@@ -0,0 +1,26 @@
+<template>
+  <div class="about">
+    <v-card class="mx-auto" max-width="400">
+      <v-card-title>Word Scramble Game</v-card-title>
+      <v-card-text>
+        <p>
```
Comment (mmwoodfo) [inline wordle-web/src/views/game.vue:6]: No need to wrap this in a p tag

File: wordle-web/src/views/game.vue [hunk 3]
```diff
@@ -0,0 +1,26 @@
+<template>
+  <div class="about">
```
Comment (mmwoodfo) [inline wordle-web/src/views/game.vue:2]: No need to use a div here

File: wordle-web/src/views/game.vue [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+      </v-card-text>
+      <v-card-actions>
+        <v-btn @click="$router.push('/')">Go back to Home</v-btn>
+      </v-card-actions>
+    </v-card>
+  </div>
+</template>
+
+<style>
+@media (min-width: 1024px) {
```
Comment (mmwoodfo) [inline wordle-web/src/views/game.vue:19]: What is the point of this styling?

File: wordle-web/src/views/game.vue [hunk 5]
```diff
@@ -0,0 +1,26 @@
...
+    <v-card class="mx-auto" max-width="400">
+      <v-card-title>Word Scramble Game</v-card-title>
+      <v-card-text>
+        <p>
+          Lorem ipsum dolor sit amet, consectetur adipiscing elit. Fusce tincidunt odio eget tellus
+          euismod efficitur.
+        </p>
+      </v-card-text>
+      <v-card-actions>
+        <v-btn @click="$router.push('/')">Go back to Home</v-btn>
```
Comment (mmwoodfo) [inline wordle-web/src/views/game.vue:12]:
```suggestion
        <v-btn to='/'>Go back to Home</v-btn>
```

**Pull request closed without merging**